src/main/java/com/autotune/analyzer/kruizeObject/ExperimentUseCaseType.java

```diff
@@ -39,12 +39,22 @@ public ExperimentUseCaseType(KruizeObject kruizeObject) throws Exception {
                 throw new Exception("Invalid Mode " + kruizeObject.getMode() + " for target cluster as Remote.");
             }
         } else if (kruizeObject.getTarget_cluster().equalsIgnoreCase(AnalyzerConstants.LOCAL)) {
-            if (kruizeObject.getMode().equalsIgnoreCase(AnalyzerConstants.MONITOR)) {
-                setLocal_monitoring(true);
-            } else if (kruizeObject.getMode().equalsIgnoreCase(AnalyzerConstants.EXPERIMENT)) {
-                setLocal_experiment(true);
-            } else {
-                throw new Exception("Invalid Mode " + kruizeObject.getMode() + " for target cluster as Local.");
+            switch (kruizeObject.getMode().toLowerCase()) {
+                case AnalyzerConstants.MONITOR:
+                    setLocal_monitoring(true);
+                    break;
+
+                case AnalyzerConstants.EXPERIMENT:
+                    setLocal_experiment(true);
+                    break;
+
+                case AnalyzerConstants.RECREATE:
+                case AnalyzerConstants.AUTO:
```

Comment:
Please combine this with MONITOR as currently they are all the same

Reply:
Sure dino! I have updated the code to use `switch` `case` block and combined the `RECREATE` and `AUTO` cases with the `MONITOR` case.